ccd-definition/CaseEventToComplexTypes/BreathingSpaceLRspec-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "ID": "BreathingSpaceStartInfo",
```

Comment:
What about permissions for the fields? do they already exist?

Reply:
Permission for the field is added already, the name for the related field should be "enterBreathing"